plataform_front_end/public/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+
+    <link href="/home/quemia/Documentos/Projetos/Homer-Project/plataform_front_end/src/components/loader/loader.css" rel="stylesheet">
```

Comment:
It will only work on @Quemia machine. If this is a global css you can import this styles into the `App.js` component:
```jsx
import 'components/loader/loader.css'

export default function App {...
```

But I don't think it should be a global style, so I would create a component `loader.js` and import as:
```jsx
import './loader.css'

const Loader = ( ) => (
  <div>
  
  </div>
)
```

Let me know if you girls have any doubt, we can talk about it and I can explain it or give some examples.

Reply:
This style is already imported into: https://github.com/maclararose/Homer-Project/pull/1/files#diff-07b20d81ddd25a326fa280a2546fb564d51a8326550fc7b076239430d6010447R2

I think you can remove this link and it should be good.